### ISSUE TITLE
Edit Effect Modal Component Width [Feature Request] 

### ISSUE THREAD

**Issue by ballzzpta** [crowbartools/Firebot#2246]
**Describe the solution you'd like**
Enter a clear and concise description of what you want to happen.
The ability to not only adjust the height of the component but to also have the ability to adjust the width of the component as well.
**Additional context**

![image](https://github.com/crowbartools/Firebot/assets/93107320/6b87bc1c-ef2c-4c03-bf53-1fc47f104118)
It makes it really hard to take advantage of the built in features as the box is a rather small box to code in especially considering I have the ability to stretch Firebot across all 3 of my monitors but am unable to change the width of that window

**Comment by ericwebster** [comment]
yeah the UI boxes inside of Firebot being able widen them would be fantastic. +1

**Comment by CKY-** [comment]
This Would Be AMAZING!!!
